.github/workflows/main.yml

```
@@ -114,7 +114,7 @@ jobs:
       - uses: actions/checkout@v3
       - run: pytest
       - run: flake8
-      - run: /github/home/.local/bin/ruff .
+      - run: /github/home/.local/bin/ruff check.
```

There should be a space between the `check` parameter and the path:

```suggestion
      - run: /github/home/.local/bin/ruff check .
```